UiPath/Script.Activities
# OSAC 128 - Prepare Credentials activities for localization

Description
Prepare Credentials activities for localization
How to test
Change Studio language and check translations for Credentials activities
Jira
[OSAC-128](https://uipath.atlassian.net/browse/OSAC-128)
Label:
24S
